design/fips-compliance.md

```diff
@@ -0,0 +1,199 @@
+# FIPS compliance for conjur-authn-k8s-client
+
+## Background
+
+The National Institute of Standards and Technology (NIST) issued the [FIPS
+ 140-2](https://csrc.nist.gov/publications/detail/fips/140/2/final) 
+Publication Series to coordinate the requirements and standards for cryptography 
+modules that include both hardware and software components.
+
+By making the conjur-authn-k8s-client FIPS compliant, our costumer will gain a 
+cryptographic standard to protect their unclassified, but sensitive data.
+
+## Useful Links
+
+| **Name**                        | **Link**                                            |
+|---------------------------------|-----------------------------------------------------|
+| FIPS compliant crypto in golang | https://kupczynski.info/2019/12/15/fips-golang.html |
+
+## Solution
+
+We looked into 3 options for making out product FIPS compliant:
```

Comment:
making out -> "to achieve produce FIPS compliancy"

Reply:
oh it was supposed to be `making our`